accounts/models.py

```diff
@@ -13,7 +13,7 @@ class CustomUser(AbstractBaseUser, PermissionsMixin):
     date_joined = models.DateTimeField(default=timezone.now)
 
```

Comment:
Include the fields in the Custom user model. (first_name and last_name fields)
Check on your variable naming. (first_name instead of firstname)